integration/integration_test.go

```diff
@@ -733,7 +734,7 @@ func TestTcptop(t *testing.T) {
 
 	tcptopCmd := &command{
 		name:           "Start tcptop gadget",
-		cmd:            fmt.Sprintf("$KUBECTL_GADGET tcptop --node $(kubectl get node --no-headers | cut -d' ' -f1) -n %s -p test-pod", ns),
```

Comment:
Good catch but I think I am also dealing with it in #500.

Reply:
Indeed, I should have lost it during the rebase process or maybe I only thought to deal with it.
So, thank you for dealing with it here.